PlanovacUdalosti/Program.cs

```diff
@@ -78,9 +69,19 @@ static void Main(string[] args)
             }
         }
     }
+
+    static void AddEvent(Event ev, Dictionary<DateOnly, int> stats, List<Event> events)
+    {
+        events.Add(ev);
+
+        if (stats.ContainsKey(ev.Date))
+            stats[ev.Date]++;
+        else
+            stats[ev.Date] = 1;
+    }
 }
 
-public class Event
+public class Event // Separatni soubor
```

Comment:
Obvykle doporucuji nasledovat konvenci 1 trida = 1 soubor, ve vetsich programech to hodne pomaha s citelnosti 🙂